src/content/community/releases/_index.en.md

```diff
@@ -5,6 +5,56 @@ weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
 
+## v0.15.0
+
+### New features
+
+* To be compliant with the [Kubernetes Multicluster Services specification][MCS KEP], service discovery now distributes a single aggregated
+  ServiceImport to each cluster in the exported service's namespace. Previously each cluster distributed its own ServiceImport copy that was
+  placed in the `submariner-operator` namespace.
+* Submariner can now be installed on a dual-stack Kubernetes cluster although, currently, only IPv4 addresses are supported.
+* Add support for selecting the node to schedule `submariner-operator`, `submariner-lighthouse-agent`, `submariner-lighthouse-coredns`, and
+  `submariner-networkplugin-syncer` pods using new `NodeSelector` and `Tolerations` fields on the `Submariner` and `ServiceDiscovery` specs.
+* Add a `subctl recover-broker-info` command to recover lost a `broker-info.subm` file.
+* It is now possible to customize the default TCP MSS clamping value set by Submariner also for non-Globalnet deployments.
+* The `subctl gather` command will now gather iptables logs for Calico and kindnet CNIs.
```

Comment:
Just to be consistent with L21 et al

```suggestion
* The `subctl gather` command now gathers iptables logs for Calico and kindnet CNIs.
```

Reply:
This is how it's written in v0.13.4 and v0.14.2, but I can change them all.